app/models/booking.rb

```diff
@@ -45,15 +45,19 @@ class Booking < ApplicationRecord
     where("bookings.start_date >= ? and bookings.end_date <= ?", sdate, edate)
   end)
 
+  scope :room_ids_in_booking, (lambda do
+    select("room_id")
```

Comment:
thử cách này đc ko a
```suggestion
    select(:room_id)
    .where(deleted: 0)
```